### PR TITLE
docs(zh-cn): update astro-pages

### PR DESCRIPTION
**Pull request by kecrily**
#### What kind of changes does this PR include?

- Translated content

#### Description

update astro-pages

### CODE REVIEW

**Comment by kecrily** [comment]
The page where the broken link is located also needs to be updated, and I chose to ignore it. and clean up after all updates are complete

File: src/pages/zh-cn/core-concepts/astro-pages.md [hunk 1]
```diff
@@ -4,76 +4,82 @@ title: 页面
...
 
+在你的组件模板中编写标准的 HTML [`<a>` 元素](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)，以便在页面之间进行链接。
+
 📚 阅读更多关于 [Astro 路由](/zh-cn/core-concepts/routing/)的信息。
 
-### 页面 HTML
+### Astro 页面
 
-Astro 页面必须返回完整的 `<html>...</html>` 页面响应，包括 `<head>` 和 `<body>`。（可以不包括 `<!doctype html>`，它会自动添加）
+Astro 页面使用 `.astro` 文件扩展名，支持与 [Astro 组件](/zh-cn/corecepts/astro-components/)相同的功能。
```
Comment (delucis) [inline src/pages/zh-cn/core-concepts/astro-pages.md:29]:
```suggestion
Astro 页面使用 `.astro` 文件扩展名，支持与 [Astro 组件](/zh-cn/core-concepts/astro-components/)相同的功能。
```

File: src/pages/zh-cn/core-concepts/astro-pages.md [hunk 2]
```diff
@@ -119,6 +125,25 @@ export const get: APIRoute = ({ params, request }) => {
...
+    body: buffer.toString('binary'),
+    encoding: 'binary',
+  };
+};
+
+```
+
+## HTML 页面
+
+带有 `.html` 文件扩展名的文件可以放在 `src/pages/` 中，并直接作为你网站上的页面使用。注意 [HTML 组件](/zh-cn/corecepts/astro-components/#html-components) 不支持一些关键的 Astro 功能。
```
Comment (delucis) [inline src/pages/zh-cn/core-concepts/astro-pages.md:145]:
```suggestion
带有 `.html` 文件扩展名的文件可以放在 `src/pages/` 中，并直接作为你网站上的页面使用。注意 [HTML 组件](/zh-cn/core-concepts/astro-components/#html-components) 不支持一些关键的 Astro 功能。
```

**Review by delucis — COMMENTED** [review]
A couple of these links can be fixed, but I guess you’re planning to hold this PR until `markdown-content`, `rss`, and `integrations-reference` are fully updated?

**Review by zihan-ch — APPROVED** [review]

**Comment by delucis** [comment]
Hi @kecrily! Seems like there’s still some work needed here and as there hasn’t been any recent activity, I’m going to close this PR. Please reopen or open a new PR if you want to pick this back up at any point. Thanks again for all your work! 💜

**Pull request closed without merging**